From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@SvenVandenbrande__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

AjayKumar-N committed:
(Azure Forums) The blog link is broken under the section "[Deploy from Dropbox]"

Ref : https://docs.microsoft.com/en-us/azure/app-service/app-service-deploy-content-sync
In the document referenced, the section "Deploy from Dropbox" points to a blog which has been archived and is broken. 
Should be updated with this url "https://azure.microsoft.com/en-in/blog/new-deploy-to-windows-azure-web-sites-from-dropbox" or with the latest document/blog if available.

rboucher committed:
update tutorials with checkboxes

itechedit committed:
edit pass: security-center-policies

rboucher committed:
update tutorials with checkboxes

v-nagta committed:
USER STORY 1032722 SaaS App Tutorial: Qualtrics Update

Commit:
updated file including screenshots

Commit:
fixes for Acrolinx

rboucher committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into cleanup

rboucher committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into cleanup

Commit:
updated another file including screenshots

rboucher committed:
remove author ashwink and replace with rboucher

rboucher committed:
Acrolinx fix up

rboucher committed:
spelling issue

genlin committed:
add new article

genlin committed:
update the article

genlin committed:
update the article

genlin committed:
add screenshots

genlin committed:
correct spelling error

rloutlaw committed:
requseted changes from hashicorp

chetansriv committed:
User Story 1116929: SaaS App Tutorial: IMPAC Risk Manager

mmacy committed:
[aci] usability fixes round #2

mmacy committed:
[aci] usability fixes round #2

chetansriv committed:
User Story 1116929: SaaS App Tutorial: IMPAC Risk Manager Update

chetansriv committed:
User Story 1116933: SaaS App Tutorial: Palo Alto Networks - Admin UI

chetansriv committed:
User Story 1116933: SaaS App Tutorial: Palo Alto Networks - Admin UI update

chetansriv committed:
User Story 1116935: SaaS App Tutorial: Palo Alto Networks - GlobalProtect

chetansriv committed:
User Story 1116936: SaaS App Tutorial: Palo Alto Networks - Captive Portal

chetansriv committed:
User Story 1116929: SaaS App Tutorial: IMPAC Risk Manager Update

chetansriv committed:
User Story 1116933: SaaS App Tutorial: Admin UI

alkohli committed:
Added remaining files to be retired

alkohli committed:
Fixed minor issues

chetansriv committed:
User Story 1116929: SaaS App Tutorial: IMPAC Risk Manager Update

chetansriv committed:
User Story 1116929: SaaS App Tutorial: IMPAC Risk Manager Update

chetansriv committed:
User Story 1116929: SaaS App Tutorial: IMPAC Risk Manager Update

carolinacmoravia committed:
P: Broken table due to breakline

Hello, @rayne-wiselman, 

Localization team has reported source content issue that cause localized version to have broken format compared to en-us version. 

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

mgblythe committed:
Redo Spark BI topic for new connector

alkohli committed:
fixed broken link

aashishb committed:
Corrected the iris_score file name in the service create command

mgblythe committed:
Tweaks from staging review

genlin committed:
update the article

genlin committed:
correct typo

genlin committed:
correct wording

genlin committed:
correct wording

genlin committed:
add file into toc

genlin committed:
rename the file

mmacy committed:
[aci] usability fixes

mmacy committed:
[aci] usability fixes

mgblythe committed:
Edits and image tweaks

mmacy committed:
[aci] usability fixes

alkohli committed:
fine tune

mmacy committed:
[aci] usability fixes

bandersmsft committed:
cost mgt - adding manage costs tutorial

dlepow committed:
Linux N-series: updates

Commit:
perf doc

Commit:
acrolinx

mmacy committed:
[aci] usability fixes

Commit:
style fix

mmacy committed:
[aci] usability fixes

andretms committed:
Update to new version of MSAL

mmacy committed:
[aci] minor typo fixes

Commit:
replacing obsolete customize cluster topic with links to HDIsight and Spark topics

andretms committed:
Added appsource tag to get support

alkohli committed:
acrolinx changes

dlepow committed:
additional review by Josh

maertendMSFT committed:
Adding bug for file creation in Azure drive

alkohli committed:
acrolinx changes

Commit:
added the missing items element to fix the TOC file error

alkohli committed:
acrolinx changes

maertendMSFT committed:
Updating with feedback

Commit:
classic retirement

Commit:
bullets

Commit:
fixed 2 broken links

ktoliver committed:
Merge pull request #26821 from mmacy/aci-usability-fixes-02

[aci] usability updates round #2

Commit:
Merge pull request #27018 from dlepow/fix

Linux N-series: updates

MikeRayMSFT committed:
Explain charges for SQL VM after uninstallation

https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1085278

PRMerger18 committed:
Merge pull request #27028 from andretms/JavaScript-GuidedSetup-Update-MSAL

Guided Setup update to new version of MSAL - 0.1.3

GitHubber17 committed:
Merge pull request #27023 from dkkapur/master

perf doc

PRMerger16 committed:
Merge pull request #26956 from aashishb/patch-2

Corrected the iris_score file name in the service create command

PRMerger13 committed:
Merge pull request #27039 from maertendMSFT/limitations

Adding bug for file creation in Azure drive

Commit:
classic retirement

ktoliver committed:
Merge pull request #26808 from chetansriv/impactriskmanager

User Story 1116929: SaaS App Tutorial: IMPAC Risk Manager

Commit:
fixed 2 issues from review

ktoliver committed:
Merge pull request #26835 from chetansriv/paloaltoadmin

User Story 1116933: SaaS App Tutorial: Palo Alto Networks - Admin UI

MGoedtel committed:
Updated supported OS table

hirokib committed:
fixed broken storage account

rboucher committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into cleanup

rboucher committed:
small typo

PRMerger17 committed:
Merge pull request #27041 from cynthn/classic1

Classic retirement

hirokib committed:
erroneous space

rboucher committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into cleanup

rboucher committed:
small changes

hirokib committed:
issues with aligning code snippets

hirokib committed:
update date

genlin committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ci68377

genlin committed:
update the article based on review comments

ktoliver committed:
Merge pull request #26844 from chetansriv/paloaltoglobalprotect

User Story 1116935: SaaS App Tutorial: Palo Alto Networks - GlobalPro…

ktoliver committed:
Merge pull request #26892 from chetansriv/paloaltocaptiveportal

User Story 1116936: SaaS App Tutorial: Palo Alto Networks - Captive P…

CarlRabeler committed:
Update sql-database-benchmark-overview.md

PRMerger13 committed:
Merge pull request #27059 from CarlRabeler/patch-360

Update sql-database-benchmark-overview.md

linda33wj committed:
Fix azure table linked service sample

linda33wj committed:
Refine JSON examples

dominicbetts committed:
Fix typo

rayne-wiselman committed:
move procedural docs to how to instead of conceptual

dominicbetts committed:
Add note about resources

linda33wj committed:
Fix link and trigger rebuild

linda33wj committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

rayne-wiselman committed:
Update toc.yml

veljko-msft committed:
Update log-analytics-azure-sql.md

Fixed the table formatting

PRMerger15 committed:
Merge pull request #27068 from linda33wj/master

Refine ADF connector samples

PRMerger10 committed:
Merge pull request #25825 from AjayKumar-N/patch-6

(Azure Forums) The blog link is broken under the section "[Deploy fro…

PRMerger19 committed:
Merge pull request #27064 from dominicbetts/suite-deploy-resources

Add note about resources

PRMerger13 committed:
Merge pull request #27062 from dominicbetts/suite-screenshots

Fix typo

PRMerger10 committed:
Merge pull request #27069 from veljko-msft/patch-7

Update log-analytics-azure-sql.md

Commit:
Added reference to Azure DNS

hning86 committed:
Update tutorial-classifying-iris-part-3.md

PRMerger18 committed:
Merge pull request #27051 from MGoedtel/UpdateWindowsK116

Updated supported OS table

hning86 committed:
Update tutorial-classifying-iris-part-3.md

PRMerger16 committed:
Merge pull request #27070 from jimdial/patch-26

Added reference to Azure DNS

MGoedtel committed:
updated article to improve readability

Commit:
Merge pull request #27071 from hning86/patch-17

Update tutorial-classifying-iris-part-3.md

hning86 committed:
Update tutorial-iris-azure-cli.md

MGoedtel committed:
updated link to automation article

Commit:
Merge pull request #26936 from carolinacmoravia/patch-422

P: Broken table due to breakline

Commit:
fixed formatting issue

rloutlaw committed:
update for 1.0.0-beta-2 functions support

rloutlaw committed:
update dates in metdata

rloutlaw committed:
revert incorrect terraform change to align with master

MGoedtel committed:
updated agent install section

GitHubber17 committed:
Merge pull request #27065 from rayne-wiselman/rayne-november7

move procedural docs to how to instead of conceptual

ktoliver committed:
Merge pull request #26947 from mgblythe/mb-azure-spark

Redo Spark BI topic for new connector

ktoliver committed:
Merge pull request #26175 from rboucher/cleanup

update tutorials with checkboxes

GitHubber17 committed:
Merge pull request #26642 from genlin/ci68377

troubleshoot-vm-by-use-nested-virtualization .md

alkohli committed:
deleted the old security file

MGoedtel committed:
corrected link path error

Commit:
Merge pull request #27074 from rloutlaw/update-java-functions

Update java functions to include new features added in 1.0.0-beta 2

PRMerger19 committed:
Merge pull request #27052 from hirokib/master

Fix to tutorial blob location

neilpeterson committed:
remove --admin-enabled

PRMerger17 committed:
Merge pull request #27073 from hning86/patch-18

Update tutorial-iris-azure-cli.md

ktoliver committed:
Merge pull request #26351 from craigcaseyMSFT/v-craic1031

updated file including screenshots

MGoedtel committed:
corrected link path error

tfitzmac committed:
removed broken link

PRMerger12 committed:
Merge pull request #27075 from neilpeterson/acs-aks-update

remove --admin-enabled

MikeRayMSFT committed:
Add link.

Commit:
Merge pull request #27078 from tfitzmac/1107link

removed broken link

cynthn committed:
Update virtual-machines-common-sizes-table-defs.md

MGoedtel committed:
udpdated note for agent install

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-alje committed:
Merge pull request #27081 from MicrosoftDocs/FromPublicRepo

From public repo

Lisaco88 committed:
Merge pull request #26288 from v-nagta/Qualtrics

USER STORY 1032722 SaaS App Tutorial: Qualtrics Update

PRMerger18 committed:
Merge pull request #27080 from cynthn/master

Update virtual-machines-common-sizes-table-defs.md

PRMerger15 committed:
Merge pull request #27043 from MikeRayMSFT/patch-4

Explain charges for SQL VM after uninstallation

itechedit committed:
Update security-center-policies.md

DavidBosland committed:
Removing links to articles that will be deleted.

ktoliver committed:
Merge pull request #26901 from alkohli/16Classic

Added notification for classic portal files to be retired

PRMerger18 committed:
Merge pull request #27086 from DavidBosland/master

Removing links to articles that will be deleted.

ktoliver committed:
Merge pull request #27072 from MGoedtel/UpdateLinuxAgent117

updated article to improve readability

Lisaco88 committed:
Merge pull request #27012 from bandersmsft/master

cost mgt - adding manage costs tutorial

Lisaco88 committed:
Merge pull request #27031 from bradsev/new-setup-data-science-environs

replacing obsolete customize cluster topic with links to HDInsight and…

ShawnJackson committed:
Merge pull request #26228 from itechedit/security-center-policies

edit pass: security-center-policies
